custom-targets/git-ops/quickstart/QUICKSTART.md

```diff
@@ -0,0 +1,171 @@
+# Cloud Deploy Git Deployer Quickstart
+
+This page shows you how to use Cloud Deploy to deploy to a Git Repository on `github.com` using the sample Git deployer.
+
+In this quickstart you will:
+
+1. Create a Github repository and a Personal Access Token to give a service account access to the repository.
```

Comment:
sed -i 's/Github/GitHub/g' QUICKSTART.md ../README.md

because corporate branding is important